scripts/migration/migrate_date_modified_for_existing_nodes.py

```diff
@@ -0,0 +1,46 @@
+"""
+This will add a date_modified field to all nodes.  Date_modified will be equivalent to the date of the last log.
+"""
+import sys
+import logging
+from website.app import init_app
+from website import models
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+
+logger = logging.getLogger(__name__)
+
+def date_updated(node):
+    """
+    The most recent datetime when this node was modified, based on
+    the logs.
+    """
+    try:
+        return node.logs[-1].date
+    except IndexError:
+        return node.date_created
+
+def main():
+    init_app(routes=False)
+
+    logger.warn('Date_modified field will be added to all nodes.')
+
+    for node in models.Node.find():
```

Comment:
Hm, I think we can make this resume-able by adding `Q('date_modified', 'eq', None)`. @pattisdr Can you please make this change in a follow-up PR?